backend/clickhouse/logs.go

```diff
@@ -488,8 +489,12 @@ func makeSelectBuilder(selectStr string, projectID int, params modelInputs.LogsP
 
 	filters := makeFilters(params.Query)
 
-	if len(filters.body) > 0 {
-		sb.Where("Body ILIKE" + sb.Var(filters.body))
+	for _, body := range filters.body {
+		if strings.Contains(body, "%") {
+			sb.Where("Body ILIKE" + sb.Var(body))
```

Comment:
`hasToken` can only operate like a wildcard if the search is at the beginning of the token. For other cases, we fall back to using `ILIKE`, like this test case:

https://github.com/highlight/highlight/blob/998afb633ae33ad31689b30f2da8121e4abde451/backend/clickhouse/logs_test.go#L543-L548

Reply:
should line 493 be checking for `Contains(body, "*")`?

Reply:
Good question. 

We sub the `*` for `%` earlier on:
https://github.com/highlight/highlight/blob/998afb633ae33ad31689b30f2da8121e4abde451/backend/clickhouse/logs.go#L566-L567

So that's why we check for `%` here.

But this definitely is a good call out that the logic is kind of scattered. I'll look into cleaning this up so it's more explicit about what is going on.